diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index 91312fbb6c238bb60e4ad76c2a32d850b5118e2f..b266104e4375e75818127b6dd5cb19659d25a9cf 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -781,6 +781,7 @@ static_library("ui") {
     # TODO(crbug.com/344921703): Remove circular dependencies.
     "//chrome/browser/sync",
   ]
+  public_deps += [ "//brave/browser/ui" ] allow_circular_includes_from += [ "//brave/browser/ui" ]
 
   if (enable_vr && is_win) {
     deps += [ "//chrome/browser/vr:vr_base" ]
@@ -4734,7 +4735,10 @@ static_library("ui") {
       ]
       deps += [ "//chrome/updater/app/server/win:updater_legacy_idl" ]
     } else {
-      sources += [ "webui/help/version_updater_basic.cc" ]
+      if (!is_official_build) {
+        # In official build, //brave/browser/ui handles.
+        sources += [ "webui/help/version_updater_basic.cc" ]
+      }
     }
   } else {  # 'OS!="win"
     if (toolkit_views) {
