diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index dd96aa5f6740f3ec4f69942c1aa6083928490921..0c5e714ef7c2432f9557d58c9a19d026dc21cfbc 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -904,6 +904,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
